scripts/sandbox/generate.ts

```diff
@@ -104,27 +99,13 @@ const addStorybook = async ({
   try {
     await copy(beforeDir, tmpDir);
 
-    const packageManager = JsPackageManagerFactory.getPackageManager({ force: 'yarn1' }, tmpDir);
     if (localRegistry) {
-      await withLocalRegistry({
-        packageManager,
-        action: async () => {
-          await packageManager.addPackageResolutions({
-            ...storybookVersions,
-            // Yarn1 Issue: https://github.com/storybookjs/storybook/issues/22431
-            jackspeak: '2.1.1',
-          });
-
-          await sbInit(tmpDir, env, [...flags, '--package-manager=yarn1'], debug);
-        },
-        cwd: tmpDir,
-        env,
-        debug,
-      });
```

Comment:
Instead of wrapping every sandbox generation with a `withLocalRegistry`, I wrapped the whole job in it, greatly simplifying and reducing the amount of noise in the job.